src/scripts/main.js

```diff
@@ -357,4 +357,34 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
```

Comment:
Ensure that there is an HTML element with the class 'dashboard' in your document. If this element does not exist, `document.querySelector('.dashboard')` will return null, and attempting to append children to it will result in an error.